src/views/Books/BookDetail.js

```diff
@@ -1,9 +1,11 @@
 import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import Book from '../../components/book/Book';
 import { getBookById } from '../../services/books';
 
 function BookDetail() {
-  const id = 1; // TODO: Use id from route
+  const params = useParams();
```

Comment:
Nice work utilizing the useParams() hook and accessing the correct data you needed.